sdk/core/core-util/CHANGELOG.md

```diff
@@ -1,17 +1,11 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.0.1 (2022-09-01)
```

Comment:
should this be v1.1 since it includes new features?

Reply:
Yes technically we should. I have considered this too but didn't do it as the change is a refactoring that only affects our packages. But I am fine bumping the minor version. Nofitication-hubs is released and depends on this change. It depends on core-util ^1.0.0 so still should be fine with minor version bump.